### PR TITLE
DR-1159 Add plan details into billing page with not founded plan behaviour

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
 Shows the selected price with his price and also add a behaviour of select the first plan when the plan selected it's not available.

# Changes

-  Modify Billing controller. 
-  Modify html and css.
-  Add a load function.

### CODE REVIEW

File: src/wwwroot/controllers/BillingCtrl.js [hunk 1]
```diff
@@ -21,6 +21,22 @@
...
+    var queryParams = $location.search();
+    vm.planName = queryParams['plan'];
+    function activate() {
+      var planSelected;
+      return settings.getPlansAvailable().then(function(response){
+        planSelected = response.data.items.find(function(obj){
+          return obj.name == vm.planName;
+        });
+        if (!planSelected) {
+          planSelected = response.data.items[0];
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: What do you think about to redirect to `my-plan` when the plan is not valid?

**Comment by andresmoschini** [comment]
What do you think about adding some little test?

File: src/wwwroot/partials/settings/billing.html [hunk 2]
```diff
@@ -5,6 +5,18 @@
...
   </div>
+  <div class="section--wrapper billing--plan-name-container">
+    <hr class="default"/>
+    <div>
+      <label class="uppercase">{{ 'plan_text' | translate }}</label>
+      <h4 class="special uppercase">{{vm.planName}}</h4>
+    </div>
+    <div>
+      <label class="uppercase">{{ 'price_text' | translate }}</label>
+      <h4 class="special">${{vm.planPrice}} {{ 'each_month' | translate }}</h4>
```
Comment (nreal) [inline src/wwwroot/partials/settings/billing.html]: The symbol of `$` you can be changed? because this symbol you can get in of backend

Comment (andresmoschini) [inline src/wwwroot/partials/settings/billing.html]: Good catch @nreal we should use currency field, because we will use the same in the email

Comment (mvillaseco) [inline src/wwwroot/partials/settings/billing.html]: From backend i only got the country currency identifier, the icon $ will still be there. Is that ok?

Comment (andresmoschini) [inline src/wwwroot/partials/settings/billing.html]: Currency values are [official currency names](https://en.wikipedia.org/wiki/ISO_4217), for example for USA Dollars, USD is the right name.

JP accepted to show that name in place of _dollars_ or _$_.

File: src/wwwroot/controllers/BillingCtrl.js [hunk 3]
```diff
@@ -21,6 +21,27 @@
...
+    vm.activationPromise = activate();
+
+    function activate() {
+      if (!planName) {
+        return $location.path('/settings/my-plan');
+      }
+      vm.planName = planName;
+      var planSelected;
+      return settings.getPlansAvailable().then(function(response){
+        planSelected = response.data.items.find(function(obj){
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: What do you think about to declare the variable `planSelected` directly here?

**Pull request merged**